modules/data.remote/R/MODIS_LAI_prep.R

```diff
@@ -5,21 +5,21 @@
 #' @param outdir Where the final CSV file will be stored.
 #' @param search_window search window for locate available LAI values.
 #' @param export_csv Decide if we want to export the CSV file.
+#' @param skip_high_sd if we want to skip observations with high standard error.
```

Comment:
Rather than a boolean, what if you instead passed in the SD threshold you want to use and just set the default really high (e.g. 100) so that it's essentially "off" by default. Alternatively, you could set it to the current threshold and have it "on" by default (SD of 10, which implies a CI of ~40, on a variable that goes from 0-6 is already very wide)

Reply:
I added a `sd_threshold` default by NULL, which could be used for filtering out the data.